src/org/opendatakit/briefcase/reused/http/CommonsHttp.java

```diff
@@ -39,12 +39,16 @@
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClientBuilder;
 import org.opendatakit.briefcase.reused.BriefcaseException;
+import org.opendatakit.briefcase.reused.UncheckedFiles;
 import org.opendatakit.briefcase.reused.http.response.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class CommonsHttp implements Http {
   private Executor executor;
   private final int maxConnections;
   private final BasicCookieStore cookieStore;
+  private static final Logger LOG = LoggerFactory.getLogger(CommonsHttp.class);
```

Comment:
I know this is a static member, but I believe we use `logger` as name for this elsewhere.

Reply:
Sure, it makes sense to be consistent.